qemu/tests/numa_prealloc_threads.py

```diff
@@ -46,24 +62,36 @@ def run(test, params, env):
     expected_cpu_affinity = thread_context_device.get_param("cpu-affinity")
     cpu_affinity = vm.monitor.qom_get(thread_context_device_id, "cpu-affinity")
     affinity = str(cpu_affinity[0]) + "-" + str(cpu_affinity[-1])
-    test.log.debug("The affinity: %s and the expected_cpu_affinity: %s"
-                   % (affinity, expected_cpu_affinity))
+    error_context.context("The affinity: %s and the expected_cpu_affinity: %s"
```

Comment:
Not all messages need to be saved to the context, this is a single info message.

Reply:
You can register steps in error_context as you wrote in L25~L32

Reply:
Thanks for the suggestion, I've put in context the different steps of the test, it should be easier now to track the covered points